sakora-csv-impl/impl/src/java/net/unicon/sakora/impl/csv/CsvAcademicSessionHandler.java

```diff
@@ -122,8 +122,12 @@ protected void processInternal(CsvSyncContext context) {
              */
             log.error("SakoraCSV has no current academic sessions to process and session processing is on, this is an invalid state as it would cause all data (courses, memberships, etc.) to be disabled or removed, please check your sessions.csv file");
         }
-        // set the current academic sessions according to the incoming sessions
-		cmAdmin.setCurrentAcademicSessions(currentSessions);
+
+		if (!currentSessions.isEmpty() && !commonHandlerService.ignoreMissingSessions()) {
+			// set the current academic sessions according to the incoming sessions
+			cmAdmin.setCurrentAcademicSessions(currentSessions);
+		}
+
 		commonHandlerService.setCurrentAcademicSessions( currentSessions.toArray(new String[currentSessions.size()]) );
```

Comment:
Does this call do the same thing? The name is identical, should this also be inside the introduced conditional?

Reply:
No. I think this is what sets the sessions being run currently in sakora-csv